### PR TITLE
Stabilizes test-integration on circleCI

### LINKED ISSUE #244: Stabilize integration CI test
Screen Shot 2020-05-04 at 1 10 56 PM

We need to figure out how to get the `test-integration` to pass more reliably.

### PR DESCRIPTION
**Pull request by nikkolasg**
Fixes #244

### CODE REVIEW

File: beacon/handler.go [hunk 1]
```diff
@@ -114,7 +114,10 @@ func (h *Handler) ProcessPartialBeacon(c context.Context, p *proto.PartialBeacon
 	nextRound, _ := NextRound(h.conf.Clock.Now().Unix(), h.conf.Group.Period, h.conf.Group.GenesisTime)
 	currentRound := nextRound - 1
 
-	if p.GetRound() > currentRound {
+	// we allow one round off in the future because of small clock drifts
+	// possible, if a node receives a packet very fast just before his local
+	// clock passed to the next round
+	if p.GetRound() > nextRound {
```
Comment (willscott) [inline beacon/handler.go:120]: does this allow someone to run ahead too much? should it be bounded any more than this?

Comment (nikkolasg) [reply beacon/handler.go:120]: It only allows a node to accept a partial signature at _one_ round ahead of the "present round" given by the local clock of the node.

Comment (nikkolasg) [reply beacon/handler.go:120]: I've seen these debug messages a lot in the circleci output from the test-integration and it seems to work much more smoothly now.

Comment (nikkolasg) [reply beacon/handler.go:120]: I'm running it a few times in a row to make sure.

**Comment by nikkolasg** [comment]
It has ran 4 times in a row successfully, sounds fine to merge now :)

**Pull request merged**